projects/1.16/assets/geolosys/geolosys/lang/zh_cn.json

```
@@ -1 +1,103 @@
+  "block.geolosys.gold_ore_sample": "金样品",
+  "block.geolosys.lapis_ore_sample": "青金石样品",
+  "block.geolosys.quartz_ore_sample": "杂石英样品",
+  "block.geolosys.kimberlite_ore_sample": "金伯利岩样品",
+  "block.geolosys.beryl_ore_sample": "绿柱石样品",
+  "block.geolosys.ancient_debris_ore_sample": "古代遗迹样品",
+  "block.geolosys.nether_gold_ore_sample": "贫金样品",
+  "item.geolosys.iron_cluster": "铁簇",
+  "item.geolosys.gold_cluster": "金簇",
+  "item.geolosys.copper_cluster": "铜簇",
+  "item.geolosys.tin_cluster": "锡簇",
+  "item.geolosys.silver_cluster": "银簇",
+  "item.geolosys.lead_cluster": "铅簇",
+  "item.geolosys.aluminum_cluster": "铝簇",
+  "item.geolosys.uranium_cluster": "铀簇",
+  "item.geolosys.nickel_cluster": "镍簇",
+  "item.geolosys.platinum_cluster": "铂簇",
+  "item.geolosys.zinc_cluster": "锌簇",
+  "item.geolosys.osmium_cluster": "锇簇",
+  "item.geolosys.yellorium_cluster": "黄铀簇",
+  "item.geolosys.ancient_debris_cluster": "古代遗迹聚集",
+  "item.geolosys.nether_gold_cluster": "贫金簇",
+  "item.geolosys.copper_ingot": "铜锭",
+  "item.geolosys.tin_ingot": "锡锭",
+  "item.geolosys.silver_ingot": "银锭",
+  "item.geolosys.lead_ingot": "铅锭",
+  "item.geolosys.aluminum_ingot": "铝锭",
+  "item.geolosys.nickel_ingot": "镍锭",
+  "item.geolosys.platinum_ingot": "铂锭",
+  "item.geolosys.zinc_ingot": "锌锭",
+  "item.geolosys.aluminum_nugget": "铝块",
+  "item.geolosys.copper_nugget": "铜块",
+  "item.geolosys.lead_nugget": "铅块",
+  "item.geolosys.nickel_nugget": "镍块",
+  "item.geolosys.platinum_nugget": "铂块",
+  "item.geolosys.silver_nugget": "银块",
+  "item.geolosys.tin_nugget": "锡块",
+  "item.geolosys.zinc_nugget": "锌块",
+  "item.geolosys.peat_coal": "泥炭",
+  "item.geolosys.lignite_coal": "褐煤",
+  "item.geolosys.bituminous_coal": "烟煤",
+  "item.geolosys.anthracite_coal": "无烟煤",
+  "item.geolosys.lignite_coal_coke": "褐煤焦炭",
+  "item.geolosys.bituminous_coal_coke": "烟煤焦炭",
+  "item.geolosys.field_manual": "现场手册",
+  "item.geolosys.field_manual.landing": "你通过 Geolosys 进行勘探的指南。$(br2)$(l)$(#f00) 这本书正在进行中 $()$(br2)$(o)$(#f59b42) 记住可用的岩体，可能与 modpack 创建者列出的不同。$()",
+  "item.geolosys.prospectors_pick": "探矿者之选",
+  "geolosys.pro_pick.tooltip": "与指南针搭配时很有用！",
+  "geolosys.pro_pick.tooltip.nonefound_stone_surface": "该地区没有特殊的岩石",
+  "geolosys.pro_pick.tooltip.nonefound_surface": "该地区没有特殊的矿床",
+  "geolosys.pro_pick.tooltip.nonefound": "该方向没有特殊的矿床",
+  "geolosys.pro_pick.tooltip.found_surface": "在这个地区发现了 %s",
+  "geolosys.pro_pick.tooltip.found": "从你这里找到 %s %s",
+  "geolosys.pro_pick.tooltip.mode.ores": "§6 找矿床",
+  "geolosys.pro_pick.tooltip.mode.stones": "§b 寻找岩石矿床",
+  "geolosys.pro_pick.depth.below": "深度: -%s 米",
+  "geolosys.pro_pick.depth.above": "深度: +%s 米",
+  "geolosys.pro_pick.depth.at": "深度: 0 米",
+  "itemGroup.geolosys.name": "§8 Geolosys §r",
+  "geolosys.download.error.string": "geolosys.json 矿床的配置文件既找不到也找不能下载。\n如果你愿意，你可以在 https://raw.githubusercontent.com/oitsjustjose/Geolosys/1.14.x/geolosys_ores.json 下载文件，并手动将其放入你的配置文件夹（需要将其重命名为 \"geolosys.json\"）。"
```

|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

```suggestion
  "geolosys.download.error.string": "矿床的配置文件 geolosys.json 既无法找到也无法下载。\n如果你愿意，你可以在 https://raw.githubusercontent.com/oitsjustjose/Geolosys/1.14.x/geolosys_ores.json 处下载文件，并手动将其放入你的配置文件夹（需要将其重命名为 \"geolosys.json\"）。"
```